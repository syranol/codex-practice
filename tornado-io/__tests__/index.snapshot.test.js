const fs = require('fs');
const path = require('path');

test('index.html matches snapshot', () => {
  const html = fs.readFileSync(path.join(__dirname, '../public/index.html'), 'utf8');
  expect(html).toMatchSnapshot();
});
