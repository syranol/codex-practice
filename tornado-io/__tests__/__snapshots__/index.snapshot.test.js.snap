// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index.html matches snapshot 1`] = `
"<!doctype html>
<html>
<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no, minimal-ui">
  <title>Tornado.io</title>
  <link rel="stylesheet" href="style.css">
</head>
<body>
  <canvas id="game"></canvas>
  <div id="leaderboard"></div>
  <div id="fps"></div>
  <div id="instructions">Drag on mobile or move mouse on desktop to steer your tornado.</div>
  <script src="/socket.io/socket.io.js"></script>
  <script src="socket.js"></script>
  <script src="player.js"></script>
  <script src="game.js"></script>
</body>
</html>
"
`;
